RELEASE.md

```diff
@@ -0,0 +1,101 @@
+
+## Bump
+
+The "bump" step increases the _SemVer_ version number in the `package.json` file.
+
+* Make sure your local `main` branch is up to date.
+* Create new custom branch from `main`.
+* `cd /flight/flight-icons` or `cd /flight/ember-flight-icons`
+* Run `yarn bump` and choose the _SemVer_ version as agreed upon on the previous PR.
+  * _The `bump` command is interactive, you can move up and down with the keyboard, choose one option, and then hit "enter": the tool will automatically update the version in the `package.json` file for you._
+* Check the `git diff` for the project, you should see only the `package.json` file changed (with the new version).
+* Commit, push, open a pull request, and wait for approval.
+
+Once the PR has been approved and merged, you can finally move to the next step, the actual release.
+
+## Release
+
+The "release" step publishes the package on the npm registry, using the version declared in the `package.json` file, and [tags](https://www.atlassian.com/git/tutorials/inspecting-a-repository/git-tag) that specific release on git.
+
+_**IMPORTANT**: if you need to do some tests, use a **local** package registry (see below), don't test directly in production!_
+
+* Make sure your local `main` branch is up to date.
+* You will need a company-approved 2FA-enabled account on npm to publish (see [npm 2FA docs](https://docs.npmjs.com/configuring-two-factor-authentication) for more info).
+* `cd /flight/flight-icons` or `cd /flight/ember-flight-icons`
+* `yarn release`
+* Check the git diff, you should not see any change.
+
+**Notice**: this action will automatically:
+
+* publish the new version of the package on the [NPM registry](https://www.npmjs.com/) using the current _SemVer_ version declared in the `package.json` file (the one previously chosen in the `bump` step).
+* tag the current last commit in the `main` branch and push the tag to the git origin
+
+At this point check on npm that the package ([@hashicorp/flight-icons](https://www.npmjs.com/package/@hashicorp/flight-icons) or [hashicorp/ember-flight-icons](https://www.npmjs.com/package/@hashicorp/ember-flight-icons)) has been successfully published, and if it's so... well done! You just published your new package 🎉.
+
+🚨 **DON'T FORGET**:
+
+* if you're releasing **a new version of `@hashicorp/flight-icons`**:
+  * you may need to update also the dependency version in the `package.json` of `ember-flight-icons` (and then release a new version of it)
+  * you need to communicate to the product teams that are consuming `flight-icons` in their codebase to bump the version of the package
+* if you're releasing **a new version of `@hashicorp/ember-flight-icons`**:
+  * you need to communicate to the product teams that are consuming `ember-flight-icons` in their codebase to bump the version of the package
```

Comment:
Is this something we want to keep track of? Shouldn't we post an announcement (that all teams should see) and they can bump or use a dependabot to PR a bump for?

Reply:
Well, since for now we don't have any automatic process in place, I prefer to have it explicitly written here (so we don't risk to forget, it can happen). Once we decide and implement an automatic process, more than happy to remove it (or update the document to reflect the changes).